src/SharedPipelineComponents/DataLockEvents/SFA.DAS.Provider.Events.DataLock.IntegrationTests/Transfer/WhenDataLockEventsAreTransferred.cs

```diff
@@ -0,0 +1,38 @@
+﻿using NUnit.Framework;
+using CS.Common.SqlBulkCopyCat;
+using CS.Common.SqlBulkCopyCat.Model.Config.Builder;
+using SFA.DAS.Provider.Events.DataLock.IntegrationTests.Helpers;
+using SFA.DAS.Provider.Events.DataLock.IntegrationTests.TestContext;
+
+// this (commented out) test can be used to manually test the transer from transient db to Deds dbof DataLockEvents and related tables
+
+namespace SFA.DAS.Provider.Events.DataLock.IntegrationTests.Transfer
+{
+    class WhenDataLockEventsAreTransferred
+    {
+        //// comment out in GlobalSetup...
+        //[OneTimeSetUp]
+        //public void BeforeAllTests()
+        //{
+        //    //SetupDedsDatabase();
+        //    //SetupSubmissionTransientDatabase();
+        //    //SetupPeriodEndTransientDatabase();
+        //}
+
+        [Test]
+        [Ignore("Useful for manual testing only (without further work)")]
+        public void ThenItShouldTransfer()
+        {
+            //TestDataHelper.SubmissionCopyReferenceData();
+
+            const string mappingFilePath = @"C:\git\das-providerpayments\src\SharedPipelineComponents\DataLockEvents\DeployDataLock\copy mappings\DasDataLockEventsCopyToDedsMapping.xml";
```

Comment:
I can see an issue with this path for purposes of running across different dev environments.

Reply:
I've added a comment to that effect.